### PR TITLE
Added readme

### PR DESCRIPTION
**Pull request by truthfool**
- Steps to deploy ERP into K8s cluster
@avikganguly01

### CODE REVIEW

File: helm/steps-to-deploy/READme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Steps to Deploy ERP into K8s Cluster
+
+1. Clone git repository for helm chart: https://github.com/truthfool/openg2p-erp-docker.git
```
Comment (avikganguly01) [inline helm/steps-to-deploy/READme.md]: Shouldn't include your github repo path.

File: helm/steps-to-deploy/READme.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+2. Create persistent volume claim in cloud using :
+`kubectl apply -f pvc.yaml`
+
+
+3. Deploy odoo and postgresql pod using :
+ `helm install openg2p -f values.yaml bitnami/odoo`
```
Comment (avikganguly01) [inline helm/steps-to-deploy/READme.md]: Is there a particular path this command needs to be executed from or from the base path of cloned repository?

File: helm/steps-to-deploy/READme.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+4. Using k9s enter into openg2p-postgresql pod and grant superuser access to openg2p role.
```
Comment (avikganguly01) [inline helm/steps-to-deploy/READme.md]: Please elaborate on how.

File: helm/steps-to-deploy/READme.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+5. Login to postgres user using:
+
+`psql -h localhost -d openg2p -U postgres -p 5432`
```
Comment (avikganguly01) [inline helm/steps-to-deploy/READme.md]: By port forwarding first? Port forwarding step not mentioned anywhere.

**Review by avikganguly01 — CHANGES_REQUESTED** [review]
Wrong folder for readme. Readme should be placed at the folder from which all commands are to be executed. Maybe that's the helm folder. Github project readme should then point to that folder.

File: helm/steps-to-deploy/READme.md (hunk 1 above)
Comment (truthfool) [reply helm/steps-to-deploy/READme.md]: ok i will change it. It was previously there to merge the latest changes

File: helm/steps-to-deploy/READme.md (hunk 2 above)
Comment (truthfool) [reply helm/steps-to-deploy/READme.md]: i will change the folder.

File: helm/steps-to-deploy/READme.md (hunk 3 above)
Comment (truthfool) [reply helm/steps-to-deploy/READme.md]: ok got it

File: helm/steps-to-deploy/READme.md (hunk 4 above)
Comment (truthfool) [reply helm/steps-to-deploy/READme.md]: i will add all details

**Pull request merged**